apiserver/testing/stub_network.go

```diff
@@ -188,7 +188,7 @@ func (f *FakeSpace) Subnets() (bs []networkingcommon.BackingSubnet, err error) {
 			AvailabilityZones: zones,
 			Status:            status,
 		}
-		outputSubnets = append(outputSubnets, &FakeSubnet{Info: backing, id: strconv.Itoa(i)})
+		outputSubnets = append(outputSubnets, &FakeSubnet{Info: backing, id: f.SpaceId + strconv.Itoa(i)})
```

Comment:
Changed this to ensure that IDs are unique. Otherwise they repeat, starting at "0" for each space.

This stub networking implementation is luggage. The sooner it can be thrown away in favour of mocks, the better.